device/celestica/x86_64-cel_e1031-r0/plugins/sfputil.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+#
+# Platform-specific SFP transceiver interface for SONiC
+#
+
+try:
+    import time
+    from sonic_sfp.sfputilbase import SfpUtilBase
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+
+class SfpUtil(SfpUtilBase):
+    """Platform-specific SfpUtil class"""
+
+    PORT_START = 14
+    PORT_END = 17
+    PORTS_IN_BLOCK = 0
+
+    _port_to_eeprom_mapping = {}
+
+
+    @property
+    def port_start(self):
+        return self.PORT_START
+
+    @property
+    def port_end(self):
+        return self.PORT_END
+
+    @property
+    def qsfp_ports(self):
+        return range(0, self.PORTS_IN_BLOCK + 1)
+
+    @property
+    def port_to_eeprom_mapping(self):
+        return self._port_to_eeprom_mapping
+
+    def __init__(self):
+        # Override port_to_eeprom_mapping for class initialization
+        eeprom_path = '/sys/bus/i2c/devices/i2c-{0}/{0}-0050/eeprom'
```

Comment:
the eeprom driver only works for qsfp module, need to use oom module to support sfp module. 
oom module has been added to sonic kernel, so you can use it directly.

more information can be found at.
https://github.com/opencomputeproject/oom

Reply:
OK I will use oom driver to support qsfp and sfp module